.github/workflows/release.yaml

```diff
@@ -29,8 +29,6 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0  # Complete git history is required to generate the version from git tags.
-      - uses: canonical/action-build@v1
-        id: build
```

Comment:
Thanks, but you'll also need to:

- remove/modify anything that references `steps.build...`
- add a step to download the artefact from the check workflow

Reply:
Yeah, right.
Thanks for catching this